apps/i18n/music/en_us.json

```diff
@@ -58,6 +58,7 @@
   "chordArpeggioRandom": "Arpeggio Random",
   "chordTogether": "Together",
   "feedback": "Feedback",
+  "madeWithMusicLabOn": "Made with Music Lab on",
```

Comment:
You can remove this string since it will be added by https://github.com/code-dot-org/code-dot-org/pull/58252

Reply:
For now, you could hard code with 'Built with Music Lab' - thanks!

Reply:
As we discussed offline, I'll go ahead and push this commit.